### PR TITLE
Supress LAD Table View from ConditionSet

### LINKED ISSUE #5924: Condition Set LAD Table "view switch" isn't consistent with other view types
#### Summary
A number of objects (telemetry, imagery, more) in Open MCT allow view pivoting to the same data in different ways. Numeric telemetry for instance allows a pivot from a plot view to a table, and back. The app model is: view the same data, but in a different way. 

The Condition Set object also allows pivoting between its own native view as well as several different views - but those views are _not_ all of of the same data:

Screen Shot 2022-10-28 at 5 18 55 PM

Native view
Screen Shot 2022-10-26 at 4 09 33 PM

LAD Table view
Screen Shot 2022-10-26 at 4 18 50 PM

**Plot view and Telemetry Table views** both display a plot and a scrolling table of the CS's output value - not the constituents values - which is correct and desired. The native view shows the CS's current "output" based on whatever condition is currently matching, but the LAD Table view shows the current value(s) of the CS's constituents - these are not the same. This can especially be misleading when there's only one telemetry point in a Condition Set. One might argue that Imagery, which pivots between bitmaps and a scrolling table of URLs isn't obeying the model, but there's a one-to-one relationship between a given image and its corresponding URL, which is not the case with the CS pivot.

#### Proposed Solution
There are 2 fundamental solutions here:
1. **Don't allow this**. Remove "LAD Table" from the view options.
2. **Allow this**. Basically, set expectations for this view and allow it. I think this opens up options, and simply renaming "LAD Table" to "Child Values" or something similar in this instance might be enough.

#### Steps to Reproduce

1. In Browse mode, nav to a Condition Set and access its view switcher dropdown menu. 
1. Switch to the LAD Table view.

#### Environment

* Open MCT Version:  2.1.2
* Deployment Type:  all
* OS: any
* Browser: yes

#### Impact Check List

- [x] Data loss or **misrepresented** data?
- [ ] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [ ] Does this impact a critical component?
- [x] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### PR DESCRIPTION
**Pull request by scottbell**
Closes #5924 

### Describe your changes:
Check to see if in the LAD Table View if we're looking at a ConditionSet. If so, don't show the LAD Table View.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Review by charlesh88 — APPROVED** [review]
LGTM

File: e2e/tests/functional/plugins/conditionSet/conditionSet.e2e.spec.js [hunk 1]
```diff
@@ -199,4 +200,48 @@ test.describe('Basic Condition Set Use', () => {
...
+
+        await alphaGeneratorTreeItem.dragTo(conditionCollection);
+        await betaGeneratorTreeItem.dragTo(conditionCollection);
+
+        const saveButtonLocator = page.locator('button[title="Save"]');
+        await saveButtonLocator.click();
+        await page.getByRole('listitem', { name: 'Save and Finish Editing' }).click();
+        await page.click('button[title="Change the current view"]');
+
+        await expect(page.locator('li[role="menuitem"]:text("Lad Table")')).toBeHidden();
```
Comment (ozyx) [inline e2e/tests/functional/plugins/conditionSet/conditionSet.e2e.spec.js:243]: Let's use `getByRole` here too:

```js
await expect(page.getByRole('menuitem', { name: 'Lad Table' })).toBeHidden();
// etc.
```

File: e2e/tests/functional/plugins/conditionSet/conditionSet.e2e.spec.js [hunk 2]
```diff
@@ -199,4 +200,48 @@ test.describe('Basic Condition Set Use', () => {
...
+
+        // Expand the 'My Items' folder in the left tree
+        await page.locator('.c-tree__item__view-control.c-disclosure-triangle').first().click();
+        // Add the Alpha & Beta Sine Wave Generator to the Condition Set and save changes
+        const treePane = page.getByRole('tree', {
+            name: 'Main Tree'
+        });
+        const alphaGeneratorTreeItem = treePane.getByRole('treeitem', { name: "Alpha Sine Wave Generator"});
+        const betaGeneratorTreeItem = treePane.getByRole('treeitem', { name: "Beta Sine Wave Generator"});
+        const conditionCollection = await page.locator('#conditionCollection');
```
Comment (ozyx) [inline e2e/tests/functional/plugins/conditionSet/conditionSet.e2e.spec.js:233]: Remove unecessary `await` here

File: e2e/tests/functional/plugins/conditionSet/conditionSet.e2e.spec.js [hunk 3]
```diff
@@ -199,4 +200,48 @@ test.describe('Basic Condition Set Use', () => {
...
+        await createDomainObjectWithDefaults(page, {
+            type: 'Condition Set',
+            name: "Test Condition Set"
+        });
+
+        // Change the object to edit mode
+        await page.locator('[title="Edit"]').click();
+
+        // Expand the 'My Items' folder in the left tree
+        await page.locator('.c-tree__item__view-control.c-disclosure-triangle').first().click();
```
Comment (ozyx) [inline e2e/tests/functional/plugins/conditionSet/conditionSet.e2e.spec.js:226]: This is fine locally but might fail in our testing environment where YAMCS dictionaries appear before My Items.

One strategy to get around this without requiring the name of the "My Items" folder is to navigate to one of the created Condition Sets, and then clicking the "Show selected item in tree" target button on the tree:

```js
await page.goto(conditionSet1.url);
await page.click('button[title="Show selected item in tree"]');
```

**Review by ozyx — CHANGES_REQUESTED** [review]
Excellent e2e test! I've left just a couple suggestions for improvement on selectors, but otherwise LGTM (looks GREAT to me 😄 )

**Pull request merged**